fsw/cfe-core/unit-test/sb_UT.c

```diff
@@ -1,5 +1,5 @@
 /*
-**  GSC-18128-1, "Core Flight Executive Version 6.7"
+**  GSC-18128-1, "Core Flight Executive version 6.7"
```

Comment:
Recommend avoiding changes here

Reply:
GAH, I thought I caught that fat-finger error...I've fixed, squashed, and re-pushed.